### PR TITLE
(doc) Cloud MVP documentation 

### PR DESCRIPTION
**Pull request by msudberg**
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/6226)

### CODE REVIEW

File: doc/user/content/cloud/account-limits.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+{{< cloud-notice >}}
+
+For the most part, Materialize Cloud offers the same functionality as the installed version. The major exceptions are:
+
+* Materialize Cloud doesn't support using files as data sources, only streaming sources like Kafka.
+* By default, Materialize Cloud only permits two deployments. If you need more than that, [let us know](../support).
+* We reserve the right to terminate your deployment. This may happen after prolonged inactivity or if we need to upgrade Materialize Cloud. Catalog items persist across sessions.
+* Right now, we only support one user per account. If you'd like multiple users to access your Cloud console, you'll need to share the login. Alternatively, any user can connect to a Cloud deployment as you share your psql connection string. <!-- Is the last sentence correct? -->
```
Comment (benesch) [inline doc/user/content/cloud/account-limits.md:17]: (You have to share the certificates and the psql connection string, but broadly correct.)

File: doc/user/content/cloud/sql-language-drivers.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+menu:
+  main:
+    parent: "cloud"
+    weight: 5
+---
+
+
+## Connect a SQL driver to Materialize Cloud
+
+<!-- Should we actually just move this under the general SQL section? -->
```
Comment (benesch) [inline doc/user/content/cloud/sql-language-drivers.md:13]: Yes please!

File: doc/user/content/cloud/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+description: "Troubleshoot issues with Materialize Cloud."
+menu:
+  main:
+    parent: "cloud"
+    weight: 9
+---
+
+{{< cloud-notice >}}
+
+<!-- Do we have a standard set of troubleshooting questions or things we expect people to run into trouble with? If not, I will omit this page for the time being. -->
```
Comment (benesch) [inline doc/user/content/cloud/troubleshooting.md:12]: OOMs, probably, but we don't have a great story around that right now.

**Review by benesch — COMMENTED** [review]
This seems like a great start!

File: doc/user/content/cloud/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Sign up at [https://cloud.materialize.com](https://cloud.materialize.com#signup).
+
+## Create and connect to a Materialize Cloud deployment
+
+Once you [sign up](https://cloud.materialize.com/signup) for Materialize Cloud and [log in](https://cloud.materialize.com), you use the Cloud management screen to create, upgrade, or destroy deployments, and to obtain the TLS CA you need to install on your local machine.
+
+By default, you can create up to two deployments. If you're interested in more, [let us know](../support).
+
+1. Click **Create deployment**. A deployment is created with an assigned name and hostname.
```
Comment (sploiselle) [inline doc/user/content/cloud/quickstart.md:31]: Before this gets merged, can you do a last pass over the docs for passive voice? I think really any alternative is better: "We'll create a deployment...", "You'll see a deployment...", "You'll know your deployment is ready when..." etc. etc.

File: doc/user/content/cloud/security.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+description: "Learn about Materialize Cloud security options."
+menu:
+  main:
+    parent: "cloud"
+    weight: 7
+---
+
+{{< cloud-notice >}}
+
+Currently, Materialize Cloud secures your connections through TLS. In the future, the managed Materialize Cloud product will support IAM identity management for single sign-on, VPC, two-factor authentication, and encryption at rest.
```
[the remark this answers is not in the document]
Comment (sploiselle) [reply doc/user/content/cloud/security.md:12]: I don't think we should get into teaching people the basics of internet protocols. At most I think a link to a wiki article or the equivalent is sufficient if we think our users aren't going to be very experienced.

File: doc/user/content/cloud/account-limits.md (hunk 1 above)
Comment (msudberg) [reply doc/user/content/cloud/account-limits.md:17]: Corrected.

File: doc/user/content/cloud/troubleshooting.md (hunk 3 above)
Comment (msudberg) [reply doc/user/content/cloud/troubleshooting.md:12]: After discussion with @awang, I'm pulling this page for now. We can add it back once we have actual troubleshooting advice.

File: doc/user/content/cloud/quickstart.md (hunk 4 above)
Comment (msudberg) [reply doc/user/content/cloud/quickstart.md:31]: Done.

File: doc/user/content/cloud/security.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (msudberg) [reply doc/user/content/cloud/security.md:12]: @awang - I agree with Sean's suggestion to link to wiki for TLS. How do you feel about removing IAM and just saying "single sign-on integration"?

File: doc/user/content/cloud/sql-language-drivers.md (hunk 2 above)
Comment (msudberg) [reply doc/user/content/cloud/sql-language-drivers.md]: This is being tracked in #5187.

File: doc/user/content/cloud/security.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (awang) [reply doc/user/content/cloud/security.md]: Makes sense to me!  This was just a first-reaction to quickly reading the page, so please don't block on my comment!

File: doc/user/content/cloud/quickstart.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+1. Signing up for Materialize Cloud
+2. Creating a Materialize Cloud deployment
+3. Connecting to the deployment from a terminal
+4. Connecting to a data source
+5. Running sample queries
+
+## Sign up for Materialize Cloud
+
+Sign up at [https://cloud.materialize.com](https://cloud.materialize.com#signup).
```
Comment (awang) [inline doc/user/content/cloud/quickstart.md:23]: Might be redundant, but wdyt of copy/pasting "If you're interested in trying out Materialize Cloud, let us know." here as well?  To make it more obvious that we're invite-only right now

File: doc/user/content/cloud/account-limits.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+{{< cloud-notice >}}
+
+For the most part, Materialize Cloud offers the same functionality as the installed version. The major exceptions are:
+
+* Materialize Cloud doesn't support using files as data sources, only streaming sources like Kafka.
+* By default, Materialize Cloud only permits two deployments. If you need more than that, [let us know](../support).
+* We reserve the right to terminate a session on your deployment. This may happen after prolonged inactivity or if we need to upgrade Materialize Cloud. Catalog items persist across sessions.
+* Right now, we only support one user per account. If you'd like multiple users to access your Cloud console, you'll need to share the login. Alternatively, any user can connect to a Cloud deployment as you share your TLS certificates and your psql connection string.
```
Comment (awang) [inline doc/user/content/cloud/account-limits.md:17]: should we add "as long"?

> Alternatively, any user can connect to a Cloud deployment as long as you share ...

**Review by awang — DISMISSED** [review]
overall this looks great!  

i'm happy with everything except that the quickstart references pubnub, which doesn't currently seem to work.  should ship it with pubnub or kafka?

**Comment by msudberg** [comment]
> i'm happy with everything except that the quickstart references pubnub, which doesn't currently seem to work. should ship it with pubnub or kafka?

Right now cloud is 0.7.1 and PubNub is only supported as of 0.7.2. I don't know when we're going to upgrade, so I'll switch in Kafka.

File: doc/user/content/cloud/quickstart.md (hunk 6 above)
Comment (msudberg) [reply doc/user/content/cloud/quickstart.md:23]: That's in the the shared {{< cloud-notice >}} text, so it is actually on every page.

File: doc/user/content/cloud/account-limits.md (hunk 7 above)
Comment (msudberg) [reply doc/user/content/cloud/account-limits.md:17]: Whoops, thanks.

**Comment by awang** [comment]
ah, that makes sense -- 0.7.2 is in the process of getting released now, it got held up yesterday because of #6337 and #6340

**Comment by awang** [comment]
As I think about it more, PubNub is probably the best recommendation for now. Kafka is also okay to recommend, but We're still in the process of setting up Kafka monitoring so it may go down from time to time

**Comment by benesch** [comment]
Yeah it got held up specifically so we can add PubNub sources!

**Pull request merged**